src/clone-node.ts

```diff
@@ -192,77 +194,97 @@ function cloneScrollPosition<T extends HTMLElement>(
   }
 }
 
-function decorate<T extends HTMLElement>(nativeNode: T, clonedNode: T): T {
-  if (isInstanceOfElement(clonedNode, Element)) {
-    cloneCSSStyle(nativeNode, clonedNode)
-    clonePseudoElements(nativeNode, clonedNode)
-    cloneInputValue(nativeNode, clonedNode)
-    cloneSelectValue(nativeNode, clonedNode)
-    cloneScrollPosition(nativeNode, clonedNode)
-  }
-
-  return clonedNode
-}
-
-async function ensureSVGSymbols<T extends HTMLElement>(
-  clone: T,
+// TODO: optimise this and run it as a "filter" - once per <use> tag
+// https://github.com/bubkoo/html-to-image/issues/341
+// currently - it runs `clone.querySelectorAll('use')` on each node and `document.querySelector(id)` for each tag found
+// async function ensureSVGSymbols<T extends HTMLElement>(
```

Comment:
What is this dead code doing? Don't leave commented-out code that you deleted in the codebase, we have version control to find old versions of the code.